addon/components/frost-popover.js

```diff
@@ -367,8 +369,15 @@ export default Component.extend(PropTypeMixin, {
           marginLeft: -delta.left - arrowMargin,
           marginTop: -delta.top - arrowMargin
         })
+
+        if (isPresent(this.get('onDisplay'))) {
```

Comment:
@helrac This should have been

`if (isPresent(this.get('onDisplay'))) && typeOf(this.get('onDisplay')) === 'function') {`